lib/middleware/inbound.middleware.ts

```diff
@@ -25,15 +27,18 @@ export class InboundMiddleware implements NestMiddleware {
     });
   }
 
-  use (req, res, next) {
+  use(req, res, next) {
     responseTime((req, res, time) => {
       const { url, method } = req;
-      const path = normalizePath(url, this._options.withHttpMiddleware?.pathNormalizationExtraMasks, "#val");
```

Comment:
NIT - might be worth keeping the options of the current lib as  passing extra masks ( if it makes sense), and  ` `placeholder` - easy to acheieve